client/app/main/records/upload/upload.controller.js

```diff
@@ -103,7 +104,8 @@ function UploadCtrl($scope, PDF, AbsenceRecord, Auth, School, toastr) {
           absences: trigger.absences,
           student: entry.student,
           school: school._id,
-          schoolYear: schoolYear
+          schoolYear: schoolYear,
+          withdrawn: !!(prevEntry.student || {}).withdrawn
         };
```

Comment:
I'm casting it to a boolean. It is the same as doing `Bool((prevEntry.student || {}).withdrawn)`.